idb/postgres/reset_sql.go

```diff
@@ -0,0 +1,12 @@
+// Code generated from source reset.sql via go generate. DO NOT EDIT.
+
+package postgres
+
+const reset_sql = `DROP TABLE IF EXISTS account;
```

Comment:
I think there is a discrepancy here. We don't drop `txn_participation` table, but we do drop `metastate`. However, one of Will's recent migrations fixes `txn_participation`. So, if I were to run the reset function, `txn_participation` will never be fixed. I think the proper solution is to generate `txn_participation` during accounting phase instead of block import. @winder any thoughts?

Reply:
Aside from that bug, `txn_participation` isn't accounting, it's written at block load time and then we're done with it. `reset` is about re-doing everything that gets lumped into the 'accounting' pass.

Reply:
Stuff like this is why we should never document this command. In my opinion this is a development tool.

Reply:
@winder if you are fine with it, I'm also fine